sqlglot/dialects/snowflake.py

```diff
@@ -870,6 +873,7 @@ class Tokenizer(tokens.Tokenizer):
 
         KEYWORDS = {
             **tokens.Tokenizer.KEYWORDS,
+            "FILE://": TokenType.URI_START,
```

Comment:
I ended up adding this to avoid hacks. Now the `//` substring here won't be treated as a comment due to [maximal munching](https://en.wikipedia.org/wiki/Maximal_munch).